src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -39,6 +39,14 @@ export const ShowcaseLayout = ({ CAPI, NAV }: Props) => {
 
     const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
 
+    // Allow 'LayoutSlotBottom' component to render if:
+    // - a 'forceEpic' URL flag is set to true (under any circumstance)
+    // - article not set to hide reader revenue (unless 'forceEpic' is set to true)
+    // TODO: plug 'forceEpic' variable below into the URL param through frontend
+    const forceEpic = false;
+    const renderBottomSlot =
+        forceEpic || (forceEpic && !CAPI.config.shouldHideReaderRevenue);
+
```

Comment:
This probably depends on where you want to go with this (and isn't a blocker for this PR) but how would it work if this logic could lived inside `LayoutSlotBottom`?

Reply:
I think soon enough we'll actually remove this logic altogether and let the component render every time, so the Contributions service can decide what to send, or send nothing at all. For now I think we just want a basic switch to control whether or not to render the slot, until we have some targeting logic in the service itself.